### DISCUSSION TITLE
Changing the burger menu icon color

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#10633] [Q&A] [answered]
[From discord](https://discord.com/channels/372137812037730304/550416151172087808/1048346715415126188):

How do you change the the opening closing 3 horizontal lines color as well as nav items colors when they are not selected and also the back arrow button color ?

<img width="500" alt="image" src="https://user-images.githubusercontent.com/5839577/205392572-51176740-f603-4bda-b9ca-45ecbf1b6bad.png">

**Comment by jeromelaban** [comment, marked as answer]
You'll generally need to redefine the styles for the `NavigatioViewItem` and/or `NavigationView`, see:

https://github.com/unoplatform/uno/blob/a860abff881e4a88888eb1bee1be5d03ca59e9b7/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/NavigationView.xaml#L12 

or 

https://github.com/unoplatform/uno/blob/a860abff881e4a88888eb1bee1be5d03ca59e9b7/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/NavigationView.xaml#L702

We're currently working on making it easier to override colors like these from the App.xaml: 
https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/NavigationView_rs2_themeresources.xaml

**Reply by rohan215** [reply]
Could you please tell me the specific name for these styles if you happen to know them?

**Reply by jeromelaban** [reply]
It's those exact styles I pointed above, now for changing the colors, you'll need to experiment to find which is which.
